### DISCUSSION TITLE
How to add different SSL for multiple subdomains? چگونه اس‌اس‌ال را به زیردامنه‌های مختلف اضافه کنیم؟

### DISCUSSION THREAD

**Discussion by farzinsb** [hiddify/Hiddify-Manager#104] [General سایر]
hi, thanks for this great project.
I have a question, how can I add SSL for multiple subdomains? 
for example I want to use sub1.domain.site ...  with cdn ip 1.2.3.4
and sub2.domain.site ... with cdn ip  5.6.7.8

but when I add sub2.domain it still uses the ssl for the main/first subdomain  of sub1.domain.site... so it gives ssl handshake failure.

in AApanel when you add different domains you can create separate ssl for each address. I don't know how to do the same thing in hiddify.

**Comment by hiddify-com** [comment]
Please update the panel and let me know if it works or not
